marklogic-data-hub-central/ui/src/components/tiles/toolbar.tsx

```diff
@@ -12,23 +14,39 @@ interface Props {
 const Toolbar: React.FC<Props>  = (props) => {
 
     const tiles = props.tiles; // config/tiles.config.ts
+    const authorityService = useContext(AuthoritiesContext);
+    const idToAuthoritiesNeeded = {
+        load: () => authorityService.canReadLoadData(),
+        model: () => authorityService.canReadEntityModel(),
+        curate: () => authorityService.canReadMapping(),
+        run: () => authorityService.canReadFlow(),
+        explore: () => true
```

Comment:
So even a hub-central-user can access Explore tile?

Reply:
I think that makes sense for now, until we have requirements stating otherwise.

Reply:
@ryanjdew  @xnikhil08 Does this basically take care of the UI portion of https://project.marklogic.com/jira/browse/DHFPROD-4820 - i.e. don't show the Model tile unless canReadEntityModel returns true?

Reply:
@rjrudin It shows the tile still without the authority, but the tooltip states they need to contact their security admin to get access, and selecting the tile does nothing in that case.